### PR TITLE
LRQA-73362

### PR DESCRIPTION
**Pull request by weller0246**
https://issues.liferay.com/browse/LRQA-73362

### CODE REVIEW

**Comment by weller0246** [comment]
ci:test:staging:nocompile

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/exportimport/ExportImport.testcase [hunk 1]
```diff
@@ -139,6 +139,78 @@ definition {
...
+
+		LAR.importPortlet(larFileName = "custom_fields.portlet.lar");
+
+		CustomFields.viewCPIsPresent(
+			customFieldName = "Text Field",
+			resourceName = "Document");
+
+		User.logoutPG(specificURL = "http://localhost:9080");
+
+		/*
```
Comment (gaborlovasdotcom) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/exportimport/ExportImport.testcase:176]: @weller0246 Are we need/not need the commented area?

**Comment by weller0246** [comment]
ci:test:staging:nocompile

I changed the test to use virtual instance instead of a second portal due to resource problems on CI

**Comment by weller0246** [comment]
ci:test:staging:nocompile

**Comment by weller0246** [comment]
ci:test:staging:nocompile

**Comment by weller0246** [comment]
ci:test:staging:nocompile

**Comment by weller0246** [comment]
ci:test:staging:nocompile

**Pull request closed without merging**